task_2.py

```diff
@@ -0,0 +1,16 @@
+class Tester:
+
+    def __init__(self, name, deadline=True):
```

Comment:
и свойств тоже надо убрать
```suggestion
    def __init__(self, name):
```